haystack/components/extractors/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .named_entity_extractor import NamedEntityAnnotation, NamedEntityExtractor, NamedEntityExtractorBackend
```

Comment:
We usually use the full absolute path here instead of this relative.

Reply:
Switched to absolute paths. Is this a question of style or function? I have rarely come across libraries using absolute paths in submodules.

Reply:
It's about style, developed after our linting tools more than after an explicit decision we made. We initially banned the usage of complex relative imports through flake8/ruff, and while relative imports to siblings are accepted in PEP8 (like the one here), to avoid explaining the difference we just went with "always absolute". If we find the time to define a better style (e.g. admitting relative imports to siblings) and configure our tooling accordingly, I would be supportive of the initiative!